### ISSUE TITLE
omd-5.20-labs-edition on RHEL 9.2 - omd command not working as systemd service

### ISSUE THREAD

**Issue by martinjindra67** [ConSol/omd#172]
Tried to install omd-5.20-labs-edition on RHEL 9.2. Unfortunately `omd` command does not work, when called via systemd. `omd` called via commandline works.

In journal following messages can be found:
```
Nov 13 14:39:20 v-rhtest-02.somedomain.loc omd[1250]: Starting npcd.............ERROR
Nov 13 14:39:20 v-rhtest-02.somedomain.loc omd[1867]: Starting naemon...OK
Nov 13 14:39:20 v-rhtest-02.somedomain.loc omd[1901]: Starting apache...OK
Nov 13 14:39:20 v-rhtest-02.somedomain.loc crontab[1942]: (test01) REPLACE (test01)
Nov 13 14:39:20 v-rhtest-02.somedomain.loc omd[1935]: Starting crontab...OK
Nov 13 14:39:20 v-rhtest-02.somedomain.loc omd[1114]: OK
Nov 13 14:39:20 v-rhtest-02.somedomain.loc systemd[1]: omd.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Nov 13 14:39:20 v-rhtest-02.somedomain.loc systemd[1]: omd.service: Failed with result 'exit-code'.
Nov 13 14:39:20 v-rhtest-02.somedomain.loc systemd[1]: Failed to start OMD.
Nov 13 14:39:20 v-rhtest-02.somedomain.loc systemd[1]: omd.service: Consumed 12.061s CPU time.
```
On RHEL 8 everthing works fine.

Quick and very dirty workaround was, to put the line `/usr/bin/omd start` into `/etc/rc.d/rc.local`, to get omd started at boot time. Nevertheless the omd systemd service shows a failed service.

**Comment by sni** [comment]
Thanks for your report, this has been fixed already in 4ccf937f9068c0b9197b0d633dfcb618f94a923b
So it should be fine again with the next release.
You could try the patch and see if it really solves the issue for you if you like, it's only one extra line plus comments. Just make sure you add the lines exactly as in the patch, otherwise you will get conflicts with the next update.

**Comment by martinjindra67** [comment]
Thank you for your quick response!

We checked an alternate solution, which also seems to work fine. We altered the entry `LimitNOFILE=infinity` to `LimitNOFILE=1024` in the omd.service unitfile. Of course, we can't predict, if this would have other implications to omd.

Cheers
Martin
